packages/lib-webpack/src/webpack/GenerateManifestPlugin.ts

```diff
@@ -0,0 +1,24 @@
+import { pluginManifestFile } from '@openshift/dynamic-plugin-sdk';
+import type { PluginRuntimeManifest } from '@openshift/dynamic-plugin-sdk';
+import * as webpack from 'webpack';
+
+export class GenerateManifestPlugin implements webpack.WebpackPluginInstance {
+  constructor(private readonly manifest: PluginRuntimeManifest) {}
```

Comment:
Do you not need to call `super()`?

Reply:
No need for `super()` since we are not extending a base class.

We're just implementing an interface `WebpackPluginInstance` here.

Reply:
DOH, `implements` not `extends` .. gosh I gotta pay more attention